### PR TITLE
Create LICENSE

### LINKED ISSUE #18: Add software license

### PR DESCRIPTION
**Pull request by cmungall**
Fixes #18

### CODE REVIEW

**Comment by cthoyt** [comment]
Please also include this in the pyproject.toml as well

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (kltm) [inline LICENSE:189]: Date an copyright holder needs to be filled out

**Review by kltm — CHANGES_REQUESTED** [review]
Noting use of Apache 2.0, an open and permissive license with patent protection clauses.

**Comment by sierra-moxon** [comment]
fixed according to reviews.

**Review by sierra-moxon — APPROVED** [review]

**Review by caufieldjh — APPROVED** [review]

**Pull request merged**